### PR TITLE
Fix in build docker and gulpfile

### PR DESCRIPTION
**Pull request by IgnacioRodrigues**
fix(build): delete watching change in build-w-docker and change task to run gulpfile correctly

### CODE REVIEW

File: build-w-docker.sh [hunk 1]
```diff
@@ -13,13 +13,11 @@ cd $(dirname $0)
...
 export MSYS2_ARG_CONV_EXCL="*"
 
-rm -rf dev
 rm -rf dist
 
 docker build --pull -t doppler-ui-system-source .
 docker run --rm \
     -v `pwd`/dist:/work/dist \
     -p 3500:3500 \
-    -e CHOKIDAR_USEPOLLING=true \
```
Comment (cbernat) [inline build-w-docker.sh]: 💯

File: gulpfile.js [hunk 2]
```diff
@@ -96,6 +96,22 @@ gulp.task('sass:build', ['webfont'], function() {
     .pipe(gulp.dest(config.folderDist.css));
 });
 
+gulp.task('sass:dist', ['webfont'], function() {
```
Comment (cbernat) [inline gulpfile.js]: 👍

**Comment by cbernat** [comment]
Great!

**Comment by andresmoschini** [comment]
❤️ 🚀

**Pull request merged**